noir-projects/aztec-nr/aztec/src/encrypted_logs/log_assembly_strategies/default_aes128/event.nr

```diff
@@ -28,6 +20,8 @@ use dep::protocol_types::{
 };
 use std::aes128::aes128_encrypt;
 
+global HEADER_CIPHERTEXT_SIZE_IN_BYTES: u32 = 48;
```

Comment:
Copied this over here as header size diverged from the note one (no longer contains contract address).